x-pack/platform/plugins/shared/fleet/server/services/epm/elasticsearch/transform/reauthorize.ts

```diff
@@ -160,33 +166,34 @@ export async function handleTransformReauthorizeAndStart({
       (t) => t.order,
     ]);
 
-    for (const { transformId, ...meta } of sortedTransformsMetadata) {
+    for (const { transformId, unattended, ...meta } of sortedTransformsMetadata) {
       const authorizedTransform = await reauthorizeAndStartTransform({
         esClient,
         logger,
         transformId,
         secondaryAuth,
         meta: { ...meta, last_authorized_by: username },
+        shouldStopBeforeStart: unattended,
       });
 
       authorizedTransforms.push(authorizedTransform);
     }
   } else {
     // Else, create & start all the transforms at once for speed
-    authorizedTransforms = await pMap(
-      transformsMetadata,
-      async ({ transformId, ...meta }) =>
-        reauthorizeAndStartTransform({
+    const transformsPromises = transformsMetadata.map(
+      async ({ transformId, unattended, ...meta }) => {
+        return await reauthorizeAndStartTransform({
           esClient,
           logger,
           transformId,
           secondaryAuth,
           meta: { ...meta, last_authorized_by: username },
-        }),
-      {
-        concurrency: MAX_CONCURRENT_TRANSFORMS_OPERATIONS,
+          shouldStopBeforeStart: unattended,
+        });
       }
-    ).then((results) => results.flat());
+    );
+
+    authorizedTransforms = await Promise.all(transformsPromises).then((results) => results.flat());
```

Comment:
Probably worth putting this in a `pMap` and limit the concurrency, to avoid to may concurrent requests

Reply:
@nchaulet Updated here [`08c2c35` (#210217)](https://github.com/elastic/kibana/pull/210217/commits/08c2c354d9d3eb8550426403505213933a37096e)